pkg/controllers/membercluster/membercluster_controller.go

```diff
@@ -47,6 +49,9 @@ const (
 	reasonMemberClusterJoined         = "MemberClusterJoined"
 	reasonMemberClusterLeft           = "MemberClusterLeft"
 	reasonMemberClusterUnknown        = "MemberClusterUnknown"
+
+	fleetResourceLabelKey = fleetv1beta1.FleetPrefix + "isFleetResource"
+	fleetNamespaceValue   = "fleet-namespace"
```

Comment:
Hi Arvind! Maybe it would make more sense, considering the name of the label, to use `true` here.